src/IBreaker.ts

```diff
@@ -0,0 +1,201 @@
+/* Autogenerated file. Do not edit manually. */
```

Comment:
Unsure if it would actually make sense to commit these, but it would probably make life easier for consumers.

Reply:
No longer relevant to this repo but will be added in the `mento-core-ts` repo